pyproject.toml

```diff
@@ -93,6 +91,18 @@ module = [
 ignore_errors = true
 
 
+[tool.ruff]
+line-length = 120
+src = ["earthaccess", "tests"]
+exclude = ["mypy-stubs", "stubs", "typeshed"]
+
+[tool.ruff.lint]
+extend-select = ["I"]
```

Comment:
Others we might want to consider:
```yaml
extend-select = [
   "I", # isort
    "UP",  # pyupgrade
    "D",   # pydocstyle
    "ANN", # annotations
    "PTH", # use-pathlib-pth
]
```
and the corresponding
```yaml
[tool.ruff.lint.pydocstyle]
convention = "google"
```
for `"D"`, though I'm not sure if we use numpy, google, or whatever style docstrings.

Reply:
alright, opened https://github.com/nsidc/earthaccess/issues/383